modules/nodes-adding-bare-metal-host-to-cluster-using-yaml.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-yaml_{context}"]
+= Adding a bare metal host to the cluster using YAML in the web console
+
+You can add bare metal hosts to the cluster in the web console using a YAML file that describes the bare metal host.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with YAML*.
+. Copy and paste the below YAML, modifying the relevant fields with the details of your host:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-0-3-bmc-secret
+type: Opaque
+data:
+  username: <secret-username>
+  password: <secret-password>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-0-3
+spec:
+  online: true
+  bmc:
+    address: <BMC address>
+    credentialsName: openshift-worker-0-3-bmc-secret
+    disableCertificateVerification: True
+    username: <username>
+    password: <password>
+  bootMACAddress: <host boot MAC address>
+  hardwareProfile: unknown
+----
+. Select *Create* to save the YAML and create the new bare metal host.
++
+[NOTE]
+====
+This procedure creates the bare metal host. In order to create a corresponding bare metal node, scale up the number of replicas to match the number of available bare metal hosts using the `oc scale` command and the appropriate bare metal machine set.
```

Comment:
same as `Adding a bare metal host to the cluster using the web console` - what about scaling via web console ?